CHANGELOG.md

```diff
@@ -123,7 +123,7 @@ New modules
 
 * `Data.Sign.Show` to show a sign
 
-Additions to existing modules
+* `Algebra.Properties.Semigroup` adding consequences for associativity for semigroups
```

Comment:
(I made this comment on #2692 but it belongs here):

No, you need to revert this change.
The lemmas added to `Algebra.Properties.Semigroup` should each be added in an Agda code block under this line moved to be a sub-heading underneath the original section heading...